internal/workflow.go

```diff
@@ -804,6 +804,28 @@ func SideEffect(ctx Context, f func(ctx Context) interface{}) encoded.Value {
 	return encoded
 }
 
+// MutableSideEffect executes provided function once, then it look up history for value with the given id,
```

Comment:
executes provided function -> executes the provided function
then it look up history for value -> then it looks up the history for the value
given id, if there -> given id. If there